### ISSUE TITLE
files not skipped on windows

### ISSUE THREAD

**Issue by RyanHubs** [sc-forks/solidity-coverage#484]
Hi,
I got an issue when running coverage on windows.
Should I convert a windows path to posix path?

```
version: solidity-coverage@0.7.2
system: win10 x64
```
```
.solcover.js: 
module.exports = {
    skipFiles:        ["./common/EIP712.sol"],
    istanbulFolder:   "./doc/coverage/",
    istanbulReporter: ["html"],
};
```
```
code: solidity-coverage/plugins/resources/plugin.utils.js line142
function assembleTargets(config, targets=[], skipFiles=[]){
  const skipped = [];
  const filtered = [];
  const cd = config.contractsDir;

  for (let target of targets){
    if (skipFiles.includes(target)){

      skipped.push({
        canonicalPath: target,
        relativePath: toRelativePath(target, cd),
        source: loadSource(target)
      })

    } else {

      filtered.push({
        canonicalPath: target,
        relativePath: toRelativePath(target, cd),
        source: loadSource(target)
      })
    }
  }

  console.log(`targets:${JSON.stringify(targets)}`)
  console.log(`skipFiles:${JSON.stringify(skipFiles)}`)
  return {
    skipped: skipped,
    targets: filtered
  }
}
log:
targets:["C:/common/contracts/common/EIP712.sol",
             "C:/common/contracts/Migrations.sol"]
skipFiles:["C:\\common\\contracts\\common\\EIP712.sol",
               "C:\\common\\contracts\\Migrations.sol"]
```

**Comment by cgewecke** [comment]
@RyanHubs Thanks for reporting. I don't have windows box ... could you help test a solution?

If you modify this:
https://github.com/sc-forks/solidity-coverage/blob/67ca4b4d2aad60820a9e914261c5a7c87c952d6e/plugins/resources/plugin.utils.js#L135

..to this, does that fix it?
```js
const targets = shell.ls(targetsPath).map(path.normalize);
```

**Comment by RyanHubs** [comment]
Thank you for your quick reply,
I have verified that the modification can get the correct result.

**Comment by AleG94** [comment]
When is it going to be released? I'm currently stuck between this and #481.
I can't parse the file nor exclude it from coverage.

**Comment by cgewecke** [comment]
@AleG94 Apologies, I will publish again today.

**Comment by cgewecke** [comment]
Published with `0.7.3`
